evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -408,7 +408,6 @@ class DummyEstimator(Estimator):
     assert pipeline.parameters == new_pipeline.parameters
     for component, new_component in zip(pipeline._component_graph, new_pipeline._component_graph):
         assert isinstance(new_component, type(component))
-    assert pipeline.describe() == new_pipeline.describe()
```

Comment:
This actually just asserts None == None lol... also doesn't make sense to test equality via describe though (name is different), so no need for this test.